### PR TITLE
Add code for using fine-tuned models

### PR DESCRIPTION
**Pull request by cthoyt**
- [x] Upload fine-tuned models to Zenodo under CC 0 license each in their own record (@cthoyt)
   - [x] species (https://zenodo.org/record/5205530)
   - [x] location (https://zenodo.org/record/5205553)
   - [x] disease (https://zenodo.org/record/5205592)
   - [x] correct (multiclass; https://zenodo.org/record/5206139)
   - [x] correct (binary; https://zenodo.org/record/5205989)
   - [x] cell line (https://zenodo.org/record/5205915)
- [x] Automate download of fine-tuned models with PyStow (@cthoyt; see example in 660920d, done in 5dfecde)
- [x] Show how to load model into python object (@helena-balabin)
- [x] Show how to make a prediction on a given text triple, similarly to the README example that's on the general pre-trained model (@helena-balabin)

### CODE REVIEW

File: src/stonkgs/api/example.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        prediction_output: PredictionOutput = model(**data_entry, return_dict=True)
+        probabilities.append(softmax(prediction_output.logits)[0].tolist())
+        raw_results.append(prediction_output)
+
+    # Save as pickle
+    with open(os.path.join(OUTPUT_DIR, 'species_predictions.pkl'), 'wb') as file:
+        pickle.dump(raw_results, file, protocol=pickle.HIGHEST_PROTOCOL)
+
+    # Save as a dataframe
+    probabilities_df = pd.DataFrame(probabilities, columns=["mouse", "rat", "human"])
```
Comment (cthoyt) [inline src/stonkgs/api/example.py:93]: can we generate the column list programatically?

File: src/stonkgs/api/example.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        embedding_name_to_vector_path=ensure_embeddings(),
+        embedding_name_to_random_walk_path=ensure_walks(),
+    )[["input_ids", "attention_mask", "token_type_ids"]]
+    click.echo(f"done processing df for embeddings after {time.time() - t:.2f} seconds")
+
+    dataset = Dataset.from_pandas(preprocessed_df)
+    dataset.set_format("torch")
+
+    # Initialize Softmax to process the logits predictions later on
+    softmax = torch.nn.Softmax(dim=1)
```
Comment (cthoyt) [inline src/stonkgs/api/example.py:37]: @helena-balabin what's the advantage of using the modular form versus just applying the functional form?

File: src/stonkgs/api/example.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    raw_results = []
+    probabilities = []
+    for idx, _ in tqdm(preprocessed_df.iterrows(), desc="Inferring"):
+        # Process each row at once
+        data_entry = {
+            key: torch.tensor([value]) for key, value in dict(preprocessed_df.iloc[idx]).items()
+        }
+
+        prediction_output: PredictionOutput = model(**data_entry, return_dict=True)
+        probabilities.append(softmax(prediction_output.logits)[0].tolist())
```
Comment (cthoyt) [inline src/stonkgs/api/example.py:49]: how am I supposed to interpret the results coming out of this? These numbers aren't between 0 and 1 so they're not really probabilities:

Screen Shot 2021-08-17 at 09 05 19

File: src/stonkgs/api/example.py (hunk 1 above)
Comment (helena-balabin) [reply src/stonkgs/api/example.py:98]: Yes, the label mapping is something I def want to improve once I get the time to do so (see Issue #5).

File: src/stonkgs/api/example.py (hunk 2 above)
Comment (helena-balabin) [reply src/stonkgs/api/example.py:37]: There's no specific reason other than my (bad) habits, you're right, the functional form would have totally been sufficient in this case.

File: src/stonkgs/api/example.py (hunk 1 above)
Comment (cthoyt) [reply src/stonkgs/api/example.py:97]: I looked into it and the code doesn't seem to save this ever

**Comment by ddomingof** [comment]
@cthoyt ready to merge?

**Comment by cthoyt** [comment]
Yup was just waiting on CI!
